### DISCUSSION TITLE
Compact mode for live timing screen

### DISCUSSION THREAD

**Discussion by meph0** [f1multiviewer/issue-tracker#264] [Feature requests] [answered]
Would love a compact version of the live timing screen with all information on a single line. Basically the official data channel, with mini sectors during the race and without the real-time speed/gear indicator.

**Comment by natgracing** [comment]
yep this and ideally some customization too, just like the stream overlays I use for twitch streams (race mode & quali mode)
![quali](https://user-images.githubusercontent.com/128322850/226206040-3a2e888a-c3a1-4201-92d2-4bdc398d7737.JPG)
![race](https://user-images.githubusercontent.com/128322850/226206045-513da50d-3c9a-4032-9c24-1182418fa4bf.JPG)

**Comment by jeppeolesen** [comment, marked as answer]
This is possible with the MultiViewer custom live timing layouts. Click the edit button on the top right of the live timing window and you can set it up how you want with a bunch of different options available.

![Screenshot 2025-03-23 at 01 45 19](https://github.com/user-attachments/assets/4da1f0ed-60fd-4e5d-9cf2-ff6f6878fbdd)

**Comment by meph0** [comment]
The problem with the current live timing layout is that a single row takes up way too much vertical space. You could fit your screenshots in half the vertical space without a loss of data. It must become more compact and with a smaller font size to fit more data, so I can make the live timing window smaller and have it as one of multiple windows on a single monitor.

You can downscale the official data channel quite a bit while the data remains readable. Not so with the live timing window, it doesn't scale, it just gets scrollbars...

**Reply by jeppeolesen** [reply]
Ah, gotcha.

Sounds like you'll want to vote for #194
